values/tempo/tempo.gotmpl

```diff
@@ -0,0 +1,111 @@
+{{- $v := .Values -}}
+{{- $t:= $v.apps.tempo }}
+{{- $st := $t.storage }}
+
+
+fullnameOverride: tempo
+
+ingester:
+  resources: {{- $t.resources.ingester | toYaml | nindent 4 }}
+  {{- if eq $st.type "filesystem" }}
+  persistence:
+    enabled: true
+    size: {{ $t.persistence.ingester.size }} # default to 10Gi
+    storageClass: null
+  {{- end }}
+
+distributor:
+  resources: {{- $t.resources.distributor | toYaml | nindent 4 }}
+  autoscaling:
+    enabled: {{ $t.autoscaling.distributor.enabled }}
+    minReplicas: {{ $t.autoscaling.distributor.minReplicas }}
+    maxReplicas: {{ $t.autoscaling.distributor.maxReplicas }}
+    targetCPUUtilizationPercentage: {{ $t.autoscaling.distributor.targetCPUUtilizationPercentage }}
+    targetMemoryUtilizationPercentage: {{ $t.autoscaling.distributor.targetMemoryUtilizationPercentage }}
+  config:
+    log_received_spans:
+      enabled: true
+      include_all_attributes: true
+      filter_by_status_error: true
+
+compactor:
+  resources: {{- $t.resources.compactor | toYaml | nindent 4 }}
+
+querier:
+  resources: {{- $t.resources.querier | toYaml | nindent 4 }}
+#   podAnnotations:
+#     policy.otomi.io/ignore: psp-host-filesystem
+  autoscaling:
+    enabled: {{ $t.autoscaling.querier.enabled }}
+    minReplicas: {{ $t.autoscaling.querier.minReplicas }}
+    maxReplicas: {{ $t.autoscaling.querier.maxReplicas }}
+    targetCPUUtilizationPercentage: {{ $t.autoscaling.querier.targetCPUUtilizationPercentage }}
+    targetMemoryUtilizationPercentage: {{ $t.autoscaling.querier.targetMemoryUtilizationPercentage }}
+
+queryFrontend:
+  resources: {{- $t.resources.queryFrontend | toYaml | nindent 4 }}
+  autoscaling:
+    enabled: {{ $t.autoscaling.queryFrontend.enabled }}
+    minReplicas: {{ $t.autoscaling.queryFrontend.minReplicas }}
+    maxReplicas: {{ $t.autoscaling.queryFrontend.maxReplicas }}
+    targetCPUUtilizationPercentage: {{ $t.autoscaling.queryFrontend.targetCPUUtilizationPercentage }}
+    targetMemoryUtilizationPercentage: {{ $t.autoscaling.queryFrontend.targetMemoryUtilizationPercentage }}
+
+memcached:
+  resources: {{- $t.resources.memcached | toYaml | nindent 4 }}
+
+metricsGenerator:
+  enabled: true
+  resources: {{- $t.resources.metricsGenerator | toYaml | nindent 4 }}
+  config:
+    storage:
+      path: /var/tempo/wal
+      wal:
+      remote_write_flush_deadline: 1m
```

Comment:
it looks like indentation error

Reply:
yes, looks like a wrong indent, but this is not the case ;-)

Reply:
hmm then what is the value for `wal` 
Can set wal to null ?

Reply:
see line 321: https://github.com/redkubes/otomi-core/blob/sr/feat/tempo/charts/tempo/values.yaml